jcli/src/jcli_app/vote/tally/mod.rs

```diff
@@ -29,16 +29,16 @@ pub enum Tally {
     ///
     /// The decrypted tally data will be printed in hexadecimal encoding
     /// on standard output.
-    DecryptVotePlan(decrypt_shares::TallyVotePlanWithAllShares),
+    DecryptResults(decrypt_shares::TallyVotePlanWithAllShares),
 }
 
 impl Tally {
     pub fn exec(self) -> Result<(), Error> {
         match self {
             Tally::DecryptionShare(cmd) => cmd.exec(),
-            Tally::VotePlanDecryptionShares(cmd) => cmd.exec(),
+            Tally::DecryptionShares(cmd) => cmd.exec(),
             Tally::Decrypt(cmd) => cmd.exec(),
-            Tally::DecryptVotePlan(cmd) => cmd.exec(),
+            Tally::DecryptResults(cmd) => cmd.exec(),
```

Comment:
I wonder if we could hide the deprecated commands from help, but still have them supported in the CLI.

Reply:
Can we remove them altogether once we have tested the new ones? I do not see much value in keeping those, and we would have to maintain more functionalities.